cmd/cluster-watchall/README.md

```diff
@@ -10,9 +10,9 @@ logical clusters, that matter here.
 This demonstrates, among other things, use of the informer on "API
 resources" in [pkg/apiwatch](../../pkg/apiwatch).
 
-The notable command line flags are as follows.
+The command line flags include all the usual ones for `kubectl`, of which a few prominent ones are shown below, plus one for configuring the metrics & debug endpoint.
```

Comment:
Is there different meaning or usage for some of the flags ? If not why do you want to explicitly list some of them? Why don't we just point to the kubectl flags doc ?

Reply:
No unusual meaning.  I added references.

I chose to highlight the ones I expect are most commonly interesting.